src/wmnds/components/content-tiles/page-contents/_page-contents.scss

```
@@ -0,0 +1,18 @@
+.wmnds-content-tile {
+  &--page-contents {
+    h2 {
```

This should be a class like
```suggestion
    &__title {
```

instead of targeting a h2 element.